migrations/20250304172534_add_repo_model.down.sql

```diff
@@ -0,0 +1,8 @@
+-- Add down migration script here
+CREATE TABLE repo_model (
+    id SERIAL PRIMARY KEY,
+    repo TEXT NOT NULL,
+    treeclosed TEXT NOT NULL CHECK (treeclosed IN ('open', 'closed')),
```

Comment:
Homu stored a numeric value P that states "closed for PRs below priority P". So I would store this column as `treeclosed INT NULL`, where `NULL` means "not closed" and a non-null value means a closed tree with the priority P.